src/actions/getCurrentVersionsByPackage.js

```diff
@@ -0,0 +1,10 @@
+export function getCurrentVersionsByPackage({ packageJson, props }) {
```

Comment:
As proposed by @abalmos, we should change this to get current version from git.getLatestTagMatchingName.

I also propose that we return a dictionary here instead of an array with `{ [packageName]: version }` so that it is easier to lookup in other actions.